config/environments/development.rb

```diff
@@ -13,16 +13,11 @@
 
   config.log_level = :debug
 
-  if Rails.root.join('tmp', 'caching-dev.txt').exist?
+  if ENV['DALLI_SERVER'].presence
```

Comment:
@MattIPv4 with `DALLI_SERVER` added to the example `.env` in the README's Docker section, I think we should be good to go. Unless folks are running the app with containers, they won't need to have this var defined in development, and the cache will default `:null_store`